tests/insert_coins.js

```diff
@@ -0,0 +1,130 @@
+const { ApiPromise, WsProvider, Keyring } = require('@polkadot/api');
+
+const keyring = new Keyring({ type: 'sr25519' });
+
+const fs = require('fs');
+
+// Create a promise API instance of the passed in node address.
+async function createPromiseApi(nodeAddress) {
+    const wsProvider = new WsProvider(nodeAddress);
+    const api = await new ApiPromise({ 
+        provider: wsProvider
+    });
+    await api.isReady;
+    return api;
+}
+const mints = [];
+
+async function main() {
+    let nodeAddress = "ws://127.0.0.1:9801";
+    
+    const api = await createPromiseApi(nodeAddress);
+    let sender = keyring.addFromMnemonic("bottom drive obey lake curtain smoke basket hold race lonely fit walk//Alice");
+    let receiver = keyring.addFromMnemonic("bottom drive obey lake curtain smoke basket hold race lonely fit walk//Bob");
+
+    const mints_file = '/home/georgi/Desktop/workspace/Manta/precomputed-15k-iterations/precomputed_mints_v3-5';
```

Comment:
Was this the script run to generate the precompute coins? Maybe add comment saying that it does that